stellar/deploy-contract/commands.js

```diff
@@ -17,6 +17,16 @@ const CONTRACT_DEPLOY_OPTIONS = {
     AxelarExample: () => [
         new Option('--use-dummy-its-address', 'use dummy its address for AxelarExample contract to test a GMP call').default(false),
     ],
+    InterchainTokenService: () => [
+        new Option(
+            '--interchain-token-version <interchainTokenVersion>',
+            'version for InterchainToken contract (defaults to InterchainTokenService version)',
```

Comment:
I recommend requiring these to avoid incorrect usage by default. It's not even guaranteed that the ITS version is a valid release version for those crates